controllers/component_dependency_update_controller.go

```diff
@@ -417,10 +417,11 @@ func (r *ComponentDependencyUpdateReconciler) handleCompletedBuild(ctx context.C
 		targets = append(targets, newTargets...)
 	}
 
+	gitRepoAtShaAnnotation := pipelineRun.Annotations[gitRepoAtShaAnnotationName]
```

Comment:
I wouldn't call it `Annotation`, that's confusing. What about `gitRepoAtShaLink`?

Reply:
ok